src/App.js

```diff
@@ -141,7 +149,7 @@ class App extends React.Component {
             label="Enter Text"
             multiline
             variant="filled"
-            onChange={this.updateInput}
+            onChange={input => this.autoTranslate(input)}
```

Comment:
Nit: Why the added function wrapper?

Reply:
In case you are wondering:

```suggestion
            onChange={this.autoTranslate}
```

Is a shortcut for what you have currently and will automatically propagate the args.

Reply:
Added because I forgot to bind the function - removed this